src/parser.coffee

```diff
@@ -181,7 +183,7 @@ class exports.Parser extends events
           # append current node onto parent's <childKey> array
           s[@options.childkey] = s[@options.childkey] or []
           # push a clone so that the node in the children array can receive the #name property while the original obj can do without it
-          objClone = Object.create(null)
+          objClone = {}
           for own key of obj
             objClone[key] = obj[key]
```

Comment:
Hi,
here also you have to filter (CVE demo starts here):
```diff
-            objClone[key] = obj[key]
+            if key != '__proto__' and key != 'constructor'
+              objClone[key] = obj[key]

Reply:
Thanks. I have updated the PR 👍